contracts/test/LightClient.t.sol

```diff
@@ -15,12 +15,17 @@ import { BN254 } from "bn254/BN254.sol";
 
 /// @dev Common helpers for LightClient tests
 contract LightClientCommonTest is Test {
-    LCTest lc;
+    LCTest public lc;
     uint32 public constant BLOCKS_PER_EPOCH_TEST = 3;
     LC.LightClientState public genesis;
     // this constant should be consistent with `hotshot_contract::light_client.rs`
     uint64 internal constant STAKE_TABLE_CAPACITY = 10;
 
+    function initLC(LC.LightClientState memory _genesis, uint32 _blocksPerEpoch) public {
+        lc = new LCTest();
+        lc.initializeForTests(_genesis, _blocksPerEpoch);
+    }
```

Comment:
let's add explanation to this function.
saying that our LightClient(Test) is written with upgradable contracts in mind, thus doesn't use normal constructor, but `initalize()` instead.
here this function works like calling constructors without going through proxy deployment etc.

^^ pls reword this

Reply:
`initalizeForTests` has been remove in 6c8e4cdeeafe5ac87bb4032176453996a2c12279  as you suggested.